docs/en/stock-management/stock.settings.md

```diff
@@ -0,0 +1,66 @@
+&raquo; [Home](../index.md) / [Inventory Management](./index.md) / Stock Settings
+
+# Stock Settings
+
+There a number of settings that affect how the BHIMA software deals with stock
+management.  To check and update the settings, one goes to the "Stock
+Settings" page (under the "Stock" menu):
+
+![Stock Settings Menu](./images/stock-settings-menu.png)
+
+If you do not see the "Stock Settings" menu item, consult your system
+adminstrator.  You need to be given permission to access that page.
+
+- To add permission for a user to access the "Stock Settings' page: Under
+  `Administration > Role Management`, click on the action menu on the right
+  end of the desired user.  In the action menu, click on `Permission`.  Scroll
+  down to "Stock Settings" (at the end of the Stock items), then click the check
+  box and the [Save] button.  The Stock Settings page should now be accessible
+  for the user.
+
+The Stock Settings Page allows access to the following items:
+
+- **Number of months for calculating the average monthly consumption** (integer)
+
+- **Default minimum number of months of security stock for depots** (integer)
+
+  - **Enable automatic confirmation of purchase orders** (yes/no).  
+	Enable automatic confirmation of purchase orders when they are created
+	without requiring manual confirmation
+
+  - **Enable permission to view depots in stock registries** (yes/no).  
+	Limit the consultation of stock registries to users according to their
+	depots
+
+  - **Enable realtime stock accounting** (yes/no).  
+	Enabling this feature will write stock movement transactions into the
+	posting journal in real time. It requires all inventory accounts to be
+	correctly configured.
+
+  - **Enable Automatic Crediting of Supplier on Stock Entry** (yes/no).  
+	Automatically adds the transactions to credit the supplier for the value of
+	stock received in a depot when entering stock from a purchase order. This is
+	only triggered if the automatic stock accounting is set as well.
```

Comment:
This text should correspond to the option above.  We need to refer to that option as either "realtime stock accounting" or "automatic stock accounting", but not both.  I think "automatic" makes most sense, since it is easier to translate into French.

Reply:
Fixed; used "automatic".